kvmagent/kvmagent/plugins/volume_backup_nbd_conversion.py

```
@@ -0,0 +1,639 @@
+    def run(self):
+        try:
+            self._export_prepare()
+            carrier_sizes = self.measure()
+            self._import_prepare(carrier_sizes)
+            image_end_offsets = self.convert()
+            return self._commit(image_end_offsets)
+        except Exception:
+            try:
+                self.forest.import_abort(self.task_uuid,
+                                         [node.old_install_path for node in self.nodes])
+            except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)